res/assets/scripts/Eventum.js

```diff
@@ -379,4 +379,22 @@ export class Eventum {
             }
         }
     }
+
+    showPreview(input_id, preview_id) {
+        var input = $(input_id);
```

Comment:
we have a coding convention:
- all jQuery objects are with `$` prefix

also, in for scripts in `res/*` use `const/let`, they are transpiled by babel later.

so:
```
const $input = $(input_id);
```